### PR TITLE
documentation/ZENKO-3427_fix_xcore+hyperdrive_refs

### PR DESCRIPTION
**Pull request by wabernat**
Fixed mentions of XCore & Hyperdrive. Changed one screencap.

### CODE REVIEW

**Comment by wabernat** [comment]
Need to know if Storage Service for Artesca offers any of the following features: 
* Additional endpoint naming requirements
* Bucket match 
* Server-side encryption
* Target helper for MPU
or any additional/novel features noteworthy for a user configuring a location. All I saw was a name and an IP address with port, and the test instance is now down.  
(See open table frames/paragraphs in Zenko/docs/docsource/operation/XDM_GUI/location_management/add_a_storage_location.rst)

File: docs/docsource/operation/XDM_GUI/location_management/add_a_storage_location.rst [hunk 1]
```diff
@@ -144,6 +145,10 @@ system.
    +---------------+----------+--------+------------+---------+
    | RING/S3C      | Yes      | \-     | \-         | \-      |
    +---------------+----------+--------+------------+---------+
+   | Storage       |          |        |            |         |
+   | Service for   |          |        |            |         |
```
Comment (terrydervaux) [inline docs/docsource/operation/XDM_GUI/location_management/add_a_storage_location.rst]:
```suggestion
   | Service for   |    Yes      |   \-    |     \-      |    \-    |
```

**Review by terrydervaux — APPROVED** [review]
LGTM

**Review by alexanderchan-scality — APPROVED** [review]

**Pull request merged**